lib/Runtime/Base/ThreadContext.h

```diff
@@ -99,24 +99,28 @@ class InterruptPoller
             void * __frameAddr = nullptr; \
             GET_CURRENT_FRAME_ID(__frameAddr); \
             Js::LeaveScriptObject<stackProbe, leaveForHost, isFPUControlRestoreNeeded> __leaveScriptObject(scriptContext, __frameAddr); \
+            BEGIN_NO_EXCEPTION; \
```

Comment:
This is probably not correct 😞 - @curtisman mentioned that there are cases when we leave script where we can still have exceptions thrown.

Reply:
Alright, reverted.

---
In reply to: [196616325](https://github.com/Microsoft/ChakraCore/pull/5335#discussion_r196616325) [](ancestors = 196616325)